### PR TITLE
Run dotnet-format on Roslyn

### PR DESCRIPTION
**Pull request by ryzngard**
More changes from running `dotnet-format` on Roslyn. 

Version of dotnet-format run: 6.0.240501

### CODE REVIEW

**Review by JoeRobich — APPROVED** [review]

**Review by davidwengier — APPROVED** [review]

File: src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs [hunk 1]
```diff
@@ -3650,7 +3650,7 @@ private void AddSynthesizedRecordMembersIfNecessary(MembersAndInitializersBuilde
             memberNames.Free();
 
             // Synthesizing non-readonly properties in struct would require changing readonly logic for PrintMembers method synthesis
-            Debug.Assert(isRecordClass || !members.Any(m => m is PropertySymbol { GetMethod.IsEffectivelyReadOnly : false }));
+            Debug.Assert(isRecordClass || !members.Any(m => m is PropertySymbol { GetMethod.IsEffectivelyReadOnly: false }));
```
Comment (JoeRobich) [inline src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs:3653]: This change was made in a separate PR that bumps the CodeStyle analyzers that are used in Roslyn (See https://github.com/dotnet/roslyn/pull/55646/files#diff-d8309181586dfce90074c20deb6e6bc4bf23ae0cf0b9c02b850ee1f9ee055091R3653)

Comment (ryzngard) [reply src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs]: Merged from main and rerun with latest

**Review by jcouv — APPROVED** [review]

**Comment by jcouv** [comment]
@ryzngard This can be merged with single compiler review since no code change.

**Pull request merged**